src/actions.js

```diff
@@ -0,0 +1,58 @@
+import axios from 'axios';
+import { store } from './App';
+
+export const setElementsData = (elementsData) => {
+  return {
+    type: 'SET_ELEMENTSDATA',
```

Comment:
Without using Flow.js, one way to avoid potential typos is to set these action types as constants and import them where you need them.

E.g. `SET_ELEMENTSDATA = 'SET_ELEMENTSDATA'` and then using `SET_ELEMENTSDATA` everywhere

Reply:
Interesting... This is only useful if we might use a specific action type more than once, right? What is a situation where we would do that?

Reply:
Discussed:
1) Create a separate file called types.js and set constants there
2) Import into actions.js and reducer.js and use it there